src/cosp.F90

```diff
@@ -351,9 +351,10 @@ function COSP_SIMULATOR(cospIN,cospgridIN,cospOUT,start_idx,stop_idx,debug)
          Lmodis_column,        & ! On/Off switch for column MODIS simulator
          Lrttov_column,        & ! On/Off switch for column RTTOV simulator (not used)
          Lradar_lidar_tcc,     & ! On/Off switch from joint Calipso/Cloudsat product
-         Lcloudsat_tcc,       & !
-         Lcloudsat_tcc2,      & !         
-         Llidar_only_freq_cloud  ! On/Off switch from joint Calipso/Cloudsat product
+         Lcloudsat_tcc,        & !
```

Comment:
Please explain _tcc and _tcc2, and ideally pick a more informative name for _tcc2.

Reply:
These parameters are not from the CloudSat/MODIS warm rain diagnostics but from CloudSat/CALIPSO products, so I leave the variable name as it is. (I just only added indents here for legibility.)